### ISSUE TITLE
Prevent methods to bypass your DNS

### ISSUE THREAD

**Issue by Poochy1130** [hagezi/dns-blocklists#2720]
Today while attending classes at a remote location my mobile data had no signal so I had to use the colleges wifi the wifi by passed my dns and I wasn't able to view anime on break cause the site was blocked the prevent method to by pass dns am I able to set that up on my iPhone or is it only for a computer and if im could you tell me the steps I would need to initiate

**Comment by hagezi** [comment]
No, not without further details about the configuration and network, but this is not the right place for such questions, as I already mentioned in another issue.

**Comment by Poochy1130** [comment]
So where can I further discuss this issue I need to safeguard my dns from being bypassed I tried to join the telegram group I was blocked a while ago could I be unblocked
